lighthouse-core/gather/computed/metrics/lantern-estimated-input-latency.js

```diff
@@ -94,7 +94,7 @@ class LanternEstimatedInputLatency extends LanternMetricArtifact {
       });
     }
 
-    return events;
```

Comment:
I don't follow the flow perfectly but it seems like the new sorting in simulator would make this one redundant. But no?
(if not, could this one be done earlier within a more common location?)

Reply:
it's redundant if the nodes all start in the order that they were queued (which used to always be true), but necessary in other cases. Even if it were true though, the fact that the `.entries` iterator on a map returns this in insertion order feels like an impl detail I wouldn't want important pieces of the codebase to rely on knowing since it bit me here once it changed slightly :/

I'm generally comfortable with the idea that consumers of a Map who need the results sorted going forward need to sort it themselves. As a safeguard though, I'm cool with *also* sorting the map before we return it based on start time. Sound good compromise?